.travis.yml

```diff
@@ -27,8 +26,9 @@ install:
   - pip install sphinx sphinx-bootstrap-theme coveralls ipython[all]==2.4.1
   - travis_retry pip install .
 script:
-  - export QIITA_CONFIG_FP=`pwd`/qiita_core/support_files/config_test_travis.cfg
-  - export MOI_CONFIG_FP=`pwd`/qiita_core/support_files/config_test_travis.cfg
+  - export MOI_CONFIG_FP=`pwd`/qiita_core/support_files/config_test.cfg
+  - if [ ${TRAVIS_PULL_REQUEST} == "false" ]; then export QIITA_CONFIG_FP=`pwd`/qiita_core/support_files/config_test_travis.cfg; fi
```

Comment:
These 2 if statement are equal. Can you just simple add both `export` commands in the same if clause?